poincare/include/poincare/unit.h

```diff
@@ -226,19 +226,19 @@ class Unit final : public Expression {
         Representative("min", "60*_s",
             Representative::Prefixable::No,
             NoPrefix),
-        Representative("h",   "60*60*_s",
+        Representative("h",   "60*_min",
             Representative::Prefixable::No,
             NoPrefix),
-        Representative("day", "24*60*60*_s",
+        Representative("day", "24*_h",
             Representative::Prefixable::No,
             NoPrefix),
-        Representative("week", "7*24*60*60*_s",
+        Representative("week", "7*_day",
             Representative::Prefixable::No,
             NoPrefix),
-        Representative("month", "30*7*24*60*60*_s",
+        Representative("month", "(365.25/12)*_day", // A month is a twelveth of a year
```

Comment:
```suggestion
        Representative("month", "(360/12)*_day", // A month is a twelveth of a year
```
Or maybe something like
```
 Representative("month", "30*_day",
```